sql/core/src/main/scala/org/apache/spark/sql/execution/SparkPlan.scala

```diff
@@ -64,6 +66,15 @@ abstract class SparkPlan extends QueryPlan[SparkPlan] with Logging with Serializ
 
   protected def sparkContext = sqlContext.sparkContext
 
+  protected def wholestageCodegenIdStr(codegenId: Option[Int]): String = {
```

Comment:
This is duplicated with https://github.com/apache/spark/pull/24759/files#diff-203ac90583cebe29a92c1d812c07f102R193

Reply:
@cloud-fan yeah.. sorry.. will remove.